### PR TITLE
Generic service

### PR DESCRIPTION
**Pull request by xhulz**
@kevinbluer as requested, QBS integration has been turned into generic integration in this sprint as we have to look into the QBS Docker issue

How to test:
In the Truffle networks:

1. Connect to network
2. Other Service
3. Name & Port

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,647 +1,668 @@
 {
```
Comment (michaeljohnbennett) [inline package.json:1]: your formatter for json is using 4 spaces, vs 2.

**Review by michaeljohnbennett — CHANGES_REQUESTED** [review]
remove all .DS_Store. this should have not made it into a commit if your .gitignore was working. 

formatter cleanups and removal of ds files and check your IDE and gitignore settings, rest looks fine.

File: package.json (hunk 1 above)
Comment (xhulz) [reply package.json:1]: So weird :/
Every time I commit something, husky has been starting the linters
Anyway, I will make this changes, thank you!

**Comment by xhulz** [comment]
Done @michaeljohnbennett! Needs your validation :)

**Review by michaeljohnbennett — APPROVED** [review]

**Comment by michaeljohnbennett** [comment]
fix the merge conflicts and I think your good to go.

**Pull request merged**